escape_roomba/thread_channel.py

```diff
@@ -179,9 +180,10 @@ async def async_maybe_create_from_origin(client, channel_id, message_id):
 
         # Work around https://github.com/Rapptz/discord.py/issues/5923
         guild, ci, mi = channel.guild, channel.id, message.id
+        type = discord.ChannelType
```

Comment:
You could do this rename at the module level, so you don't have to do it multiple times. A little jarring that here you are renaming it to `type`, but on line 90 it is renamed to `Type`.

Reply:
Changed!